Sources/main.cpp

```diff
@@ -3,6 +3,9 @@
 #include <QApplication>
 #include <QDesktopWidget>
 #include <QStyleFactory>
+#include <QtDebug>
+#include <QFile>
+#include <QTextStream>
 
```

Comment:
Overall it should be in new files (not in the `main.cpp`) - `Log.h` and `Log.cpp`.